lib/engine/game/g_1880/game.rb

```diff
@@ -375,6 +426,29 @@ def train_owner(train)
         def lessee
           current_entity
         end
+
+        def round_description(name, round_number = nil)
+          round_number ||= @round.round_num
+          description = "#{name} Round "
+
+          description += @turn.to_s unless @turn.zero?
+          description += ".#{round_number}"
+
+          description += " - Train Marker at #{@train_marker.name}" if @train_marker
+
+          description.strip
+        end
+
+        def check_for_foreign_investor_connection(corp)
+          fi = @minors.find { |m| m.shares&.first&.corporation == corp }
+          return true unless fi
```

Comment:
Why true?

Reply:
I don't think i need the check here anyways, the checking step checks that a minor exists before doing the check.